### ISSUE TITLE
Error in gsub("</", "\\u003c/", payload, fixed = TRUE) :    input string 1 is invalid UTF-8

### ISSUE THREAD

**Issue by ries9112** [r-lib/vroom#211]
Hello, I have noticed an error with the specific data that I am working with after reading it using vroom(), and it seems to be caused by some weird value somewhere in my data, but doing the exact same steps using read.csv() does **not** give an error, so I thought I would share my reproducible example here. 

I first got this error when I was testing a package I had not used before and I thought the error was with that package, so I filed and closed an issue there: https://github.com/vegawidget/vlbuildr/issues/54

I noticed the same error again while using vroom and things work fine when I replace vroom() with utils::read.csv():

```
library(pins)
library(DT)
library(vroom)

# Get data from pin
# Register with board - this used to work fine without a token, but you might need to enter yours
board_register_github(name = "github", repo = "ries9112/Pins", branch = "master", token = NULL)

# Pull data from board
cryptoData <- pin_get('cryptoData','github')

# Write to .csv to read it in and convert all data types at once
write.csv(cryptoData, 'cryptoData.csv')

# Read with read.csv adjusts all data types
cryptoData <- utils::read.csv("cryptoData.csv")

# Displaying this table works without issues!
datatable(head(cryptoData, 1000),  style = "default", 
          options = list(scrollX = TRUE))



# Now read with vroom to adjust all data types and do the same thing as before
cryptoData <- vroom::vroom("cryptoData.csv")

# This does not work!
datatable(head(cryptoData, 1000),  style = "default", 
          options = list(scrollX = TRUE))

# This works fine! Somewhere between row 10 and row 1000 there is an issue with a row
datatable(head(cryptoData, 10),  style = "default", 
          options = list(scrollX = TRUE))
```

**If you can't re-create the example, see the GIF I made showing the issue (too large to copy/paste here):** https://drive.google.com/file/d/1UKKtu0me9Ifg_rNXdB-_Mgsv3lkrvAiQ/preview

To summarize what's going on here, first using read.csv() gives no problems at all. Then doing the same steps but using vroom() instead gives the error  ` Error in gsub("</", "\\u003c/", payload, fixed = TRUE) :    input string 1 is invalid UTF-8` But when looking at a preview of only the first 10 rows instead of 1,000 things work with the object created through vroom() as well, which makes me think there is a weirdly formatted value somewhere between rows 10 and row 1,000 that is causing the issue, but I couldn't tell you specifically which value because there are 88 variables and it might take a bit more effort to narrow down the specific issue.

Once again, I'm able to work around this and I don't need a solution, but I wanted to report the issue since this seems odd to me.

**Comment by jimhester** [comment]
The most likely causes are a quote within a field, or an embedded newline in a field, with `num_threads > 1`.
